packages/table/src/headers/columnHeader.tsx

```diff
@@ -271,6 +275,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, IColumnHea
     }
 
     private handleDragSelectableSelectionEnd = () => {
+        this.activationCol = null;
```

Comment:
I realize this is probably good practice and we should keep it -- just asking from a code understanding perspective -- is this necessary? You're setting this on start of drag every time, I'm just making sure that if you _wanted_ to remove this line you could.